packages/libhdf5/meta.yaml

```diff
@@ -20,8 +20,8 @@ build:
         -DCMAKE_INSTALL_PREFIX=hdf5 \
         -DH5_HAVE_GETPWUID=0 \
         -DH5_HAVE_SIGNAL=0 \
-        -DBUILD_SHARED_LIBS=0 \
-        -DBUILD_STATIC_LIBS=1 \
+        -DBUILD_SHARED_LIBS=1 \
```

Comment:
Building shared libs is not strictly necessary, I think.  After looking over the build system in h5py, it looks like they are loading the .so file to do 3 things, and then discarding it...

1. extract version number (major, minor, release)
2. test for ```mpi``` support
3. test for ```ros3``` support